README.md

```diff
@@ -301,15 +301,33 @@ Asset information
     - `symbol` Asset identification
     - `description` Basic information of assets
     - `suggesteGasLimit` Suggested gas limit. It is like the `gasLimit` field in transaction. The default is 60000
-    - `stop` Whether to stop all actions of the asset. The default is ` false`
+    - `stop` Whether to stop all actions of the asset. The default is `false`. Set it to `true` can stop all the assets other than the query operation
```

Comment:
- `stop` Whether to freeze asset. The default is `false`. Set it to `true` will stop all actions about the asset except query operations